### PR TITLE
Added multiple section model example

### PR DESCRIPTION
**Pull request by sergdort**
#31

### CODE REVIEW

**Comment by sergdort** [comment]
Hey, guys!

I also was thinking, may be in this example we also could show how to deal with dynamic content in table view (e.g toggling switcher which should effect model change somehow, pressing button inside cell or other controls), because it pretty common use case

**Comment by sergdort** [comment]
Because lifecycle of the `UITableViewCell` is not the same as for `UIViewController` (in terms of displaying data)

File: Example/MultipleSectionModelViewController.swift [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        case .StepperableSection(title: let title, items: _):
+            return title
+        case .ToggleableSection(title: let title, items: _):
+            return title
+        }
+    }
+}
+
+extension MultipleSectionModel: SectionModelType {
+    typealias Item = Any
```
Comment (sergdort) [inline Example/MultipleSectionModelViewController.swift]: Also this looks to me little bit ugly, but don't have an idea how to make it nicer :)

**Comment by kishyr** [comment]
Thanks @sergdort! This is exactly what I was after :-)

File: Example/MultipleSectionModelViewController.swift (hunk 1 above)
Comment (farzadshbfn) [inline Example/MultipleSectionModelViewController.swift]: Why not use enum on `Item` too?

**Pull request merged**